### DISCUSSION TITLE
Feature Request: 微信小助手发送到thino的消息，能否支持多文件格式

### DISCUSSION THREAD

**Discussion by weqoocu** [Quorafind/Obsidian-Thino#582] [Polls 投票]
### Feature Requested | 请求的功能

插件中一直用多文件，通过小助手发送的消息，居然合并在一个文件中，两边体验不一致，无法使用

### Checklist | 检查清单

- [X] The feature would be useful to more users than just me. 该功能对于不仅仅是我自己的用户来说是有用的。

[a comment by an automated account is not shown]

**Comment by juestchaos** [comment]
其他有需要的同学可以讨论：
- 希望支持什么格式
- 另外这个对于同步而言会产生另一个问题，就是同步的附件，再另外一端同步时，需要放到哪里？

[a comment by an automated account is not shown]
